trimesh/path/packing.py

```diff
@@ -459,7 +459,11 @@ def rectangles(extents,
         # run a single insertion order
         # don't shuffle the first run, shuffle subsequent runs
         bounds, insert = rectangles_single(
-            extents=extents, size=size, shuffle=(i != 0))
+            extents=extents, 
+            size=size, 
+            shuffle=(i != 0),
+            rotate=rotate
```

Comment:
this parameter was not set, resulting in occasional wrong coordinates and overlaps.
![image](https://github.com/mikedh/trimesh/assets/11183241/3da6679a-2755-4820-bf62-cf1a53d01215)
(image was truncated to focus on interesting part)

Reply:
btw. some other bug that I was not able to find it causing the left most part to be unused. 
From my understanding, the `rectangles_single` function allocates space for this object marked in red [here in the code](https://github.com/mikedh/trimesh/blob/a2f89a6917d69e76914b09c7864acea3a5193f47/trimesh/path/packing.py#L208).
![image](https://github.com/mikedh/trimesh/assets/11183241/8200d673-3230-4997-a4a2-47e88f815d8d)

But then it starts the insertion with the largest texture which does not fit - so it allocates new space and inserts the largest texture _after_ the already allocated space.
But simply allocating 0 space also did not seem to solve it for me.

Reply:
If you want to try it out yourself: I used [this example mesh](https://sketchfab.com/3d-models/wall-unit-620a4629d52e458fbaa7b05160547ddb). It is a great sample for testing the scene fusion function!